### ISSUE TITLE
File not specified and Error 2

### ISSUE THREAD

**Issue by NecRaul** [MyLegGuy/legArchive#1]
I had some problems trying to use it at first because nmake command wouldn't work with it with regular cmd/PowerShell and I'd get errors running it with VS Developer Shell .  I got it working with chocolatey's make somehow but it's giving an error:
`cc -Wall   -c -o legArchive.o legArchive.c`
`process_begin: CreateProcess(NULL, cc -Wall -c -o legArchive.o legArchive.c, ...) failed.`
`make (e=2): The system cannot find the file specified.`
`make: *** [<builtin>: legArchive.o] Error 2`
I tried to tinker with it, mainly the prefix so it would find the legArchive files but that didn't really work either. Same thing happens when I run it with mingw32-make, the only difference being it tries to use windowsgarbage.c and windowsstuff.h instead. I was at the verge of giving up and tried to atleast see if the pre-release executable would do anything but it didn't.
I want to use the archive mainly to replace some audio files on android VNDS to Vita conversions. For example, Kagetsu Tohya and Tsukihime Plus+Disc's VNDS version don't have audio files innately but the scripts make use of them. Adding them before the conversion doesn't result in anything for some reason so I want to add them after converting to VNDS-Vita.

**Comment by NecRaul** [comment]
Alright, I completely forgot about this before I found another solution to the problem I was having (it was all my fault honestly, the sound files I was trying to use were encrypted in a different format than I had decrypted with so it wouldn't even open with mpv) but, trying to convert True Rememberance from VNDS to Vita (rather trying to use it in widescreen mode) didn't go so well, so I came back to try to use legArchive.
I didn't try CC=gcc make at the time, but now with Msys2 make works just fine. I don't remember but maybe I didn't have the make on the PATH environment variables. The thing about executable was also totally my fault. Opening the exe, I expected a GUI and when opened and closed simultaneously, I didn't bother doing anything else. 
![image](https://user-images.githubusercontent.com/40130066/144707882-ed8df09b-9eaf-4e6b-9953-ab12c8d5d677.png)
It might be too late, but thanks for the needed help.
